### PR TITLE
[ADD] finance/accounting: add avatax integration

### PR DESCRIPTION
**Pull request by StraubCreative**
**Overview**
New v14 module doc requested by @jorenvo 

**Summary of changes**
This PR adds new **Avatax** documentation inside **Accounting / Taxation / Taxes ** 
_This can be forward-ported beyond 14.0 to later versions_

### CODE REVIEW

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Hello @StraubCreative !

Thanks for this Accounting Pull Request :)
For apps handled by the BE team, you can just add a task in the documentation project with some indications if you have them, and ping the content writer or me.  But thank you for having done this one then ;) 

I wrote a few comments, but some need to be applied to the rest of the document (images, guilabel, sentence case etc).

- I suggested something slightly different for the configuration section.
- The credentials part could use a brief indication on how to retrieve the credentials.
- The configuration section only mentions that you need to add credentials, it doesn't explain anything about the environment, other options and buttons.
- Your text should suffice without images, and images come to help understand. It is sometimes the opposite (e.g., the part about overriding the avatax category on a product form). It's not a video made into a static page ^^
- `:guilabel:` should now be used wherever it's an UI element, be it a button, the name of a view or field etc.
- see comments about images filenames + "HD resolution" instead of "4K resolution"
- tip to also take smaller images regardless of the resolution: rescale the window so what you need fits in less width. I personally open Chrome's inspector to play with the width of the screen. + crop more! Let's just show what's relevant and that helps understanding the text. 
- the commit message doesn't need the "finance" category. However, you should add the number of the related odoo task  (the number of the task in the url) as such: 
  ```
  [ADD] accounting: avatax integration
  
  task-id 1234567
  ```
- you can also copy-paste the url of the odoo task in the first message on this Pull Request. This way, reviewers and mergers can easily update the status of the task.
- the label "new content request" is rather for issues. No need for such label anymore, but the most appropriate one would be "content improvement". But since close to 100% of the PR on this repo are content improvement, it's not really necessary.

Thanks and have a good one :)

File: content/applications/finance/accounting/taxation/taxes/avatax.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+.. _avatax/credentials:
+
+Credential configuration
+========================
+
+Avatax can be configured in :menuselection:`Accounting --> Configuration --> Settings`. If you do
+not yet have credentials, follow the **How to Get Credentials** link in the **Settings** window for
+Avatax.
+
+.. image:: avatax/avatax-01.png
```
[the remark this answers is not in the document]
Comment (StraubCreative) [reply content/applications/finance/accounting/taxation/taxes/avatax.rst:16]: Changed the filename, no problem there.
For the resolution on this I got 706x880 which is barely enough for tablet breakpoint resolution of 767px. Weight is also only 23KB. Can you clarify further here? (I thought HD screen resolution was ~1080p which this is smaller than, no?)

File: content/applications/finance/accounting/taxation/taxes/avatax.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+.. _avatax/tax-mapping:
+
+Tax Mapping
+===========
+
+The Avatax integration is available on Sale Orders and Invoices with the included Avatax fiscal
+position.
+
+Before using the integration, specify an Avatax Category on the product categories you use.
```
[the remark this answers is not in the document]
Comment (StraubCreative) [reply content/applications/finance/accounting/taxation/taxes/avatax.rst:28]: Couldn't find a "product categories" doc that was relevant here.
The seealso doc I included at the bottom was [fiscal positions](https://www.odoo.com/documentation/15.0/applications/finance/accounting/taxation/taxes/fiscal_positions.html) just like you see on TaxCloud doc. Open to suggestions if you have any!

File: content/applications/finance/accounting/taxation/taxes/avatax.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+.. image:: avatax/avatax-02.png
+   :align: center
+   :alt: Specify Avatax Category on products
+
+Product categories may be overridden or set on individual products as well.
```
[the remark this answers is not in the document]
Comment (StraubCreative) [reply content/applications/finance/accounting/taxation/taxes/avatax.rst:34]: Yea I think your suggestion makes more sense than what was originally written. Would need to get further clarification from JOV / R&D team maybe on the content expansion round.

**Comment by StraubCreative** [comment]
Hi @jcs-odoo 👋 

Please see newest revisions on [6cc1d4c](https://github.com/odoo/documentation/pull/1845/commits/6cc1d4cff256ed56c998b4171f85cb97cc07dab7).

I made a number of the changes you asked for, sticking closely to the technical stuff and not so much the content like we talked about briefly.

One change I took the liberty of making was to streamline the spelling of **Avatax** (instead of AvaTax) just to make it more consistent and that's the spelling the R&D team decided to favor anyway in the UI.

Please let me know there are any other changes you'd like me to make before we try to expand the content next. Can knock out quickly if you see anything else.

Thanks!

**Review by jcs-odoo — APPROVED** [review]
Hey @StraubCreative :)

I took the liberty to push some changes
- some typos
- line break error
- a :guilabel: instead of bold
- rescale and crop the images

Feel free to check the diff between our versions if you'd like.

Let's merge this as-is for now. We'll add more details later :)

Thanks again!

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**